### ISSUE TITLE
Inline Attachments

### ISSUE THREAD

**Issue by davidpurkiss** [nodemailer/nodemailer-sendgrid#2]
Hi

I'd like to create a PR for a fix for inline attachments. There are two issues with the current code.

- When using a content id, the `disposition` should be set to `inline`.
- The `contentId` field should be `content_id`.

Here is the updated code if you want to go ahead and make the changes

 ```javascript
case 'attachments':
{
    let attachments: any[] = source.attachments.map((entry: any) => {
        let attachment: any = {
            content: entry.content,
            filename: entry.filename,
            type: entry.contentType,
            disposition: 'attachment'
        };
        if (entry.cid) {
            // --- Begin changes ---
            attachment.content_id = entry.cid;
            attachment.disposition = 'inline';
           // --- End changes ---
        }
        return attachment;
    });

    msg.attachments = [].concat(msg.attachments || []).concat(attachments);
}
break;
```

**Comment by andris9** [comment]
Could you create a real PR for this so you would be logged as the contributor?

**Comment by davidpurkiss** [comment]
Sure, didn't realize I could create a PR across forks. PR #3 has been created.

**Comment by andris9** [comment]
Published the change as v1.0.3

**Comment by davidpurkiss** [comment]
Great! Thank you!!
